### PR TITLE
Fix batch size for latency measurement

### PR DESCRIPTION
**Pull request by macsz**
Batch size is not set properly to the runners for latency measurement. This PR addresses this issue for image classification supernets.

Changed params:

- `--batch_size` will be used for latency measurement
- `--eval_batch_size` will be used for accuracy measurement

### CODE REVIEW

**Comment by macsz** [comment]
We should have a separate batch size configuration for performance/validation measurements (no one wants to validate imagenet with bs=1).

**Comment by macsz** [comment]
Since this PR was created new search spaces have been added, and so this work has to be extended to cover them and re-tested.

File: dynast/cli.py [hunk 1]
```diff
@@ -80,6 +81,12 @@ def main():
     parser.add_argument('-d', '--device', default='cpu', type=str, help='Target device to run measurements on.')
     parser.add_argument('--num_evals', default=250, type=int, help='Total number of evaluations during search.')
     parser.add_argument('--batch_size', default=128, type=int, help='Batch size for latency measurement calculation.')
+    parser.add_argument(
+        '--test_batch_size',
+        default=None,
+        type=int,
+        help='Batch size used in accuracy evaluation process. Not used for latency measurement. If not set it will default to the same value as `batch_size` parameter.',
+    )
```
Comment (sharathns93) [inline dynast/cli.py:89]: does eval_batch_size make more sense here instead of test_batch_size?

Comment (macsz) [reply dynast/cli.py:89]: I followed OFA naming convention, but maybe OFA is not the best role model after all. Let me revert it to eval batch size.

**Pull request merged**